certbot/tests/error_handler_test.py

```diff
@@ -1,11 +1,31 @@
 """Tests for certbot.error_handler."""
+import os
 import signal
 import sys
 import unittest
+from contextlib import contextmanager
```

Comment:
nit: In general, we just import the module rather than specific methods, decorators, etc. So here, you'd want to `import contextlib` (as the first import) and change line 11 to `@contextlib.contextmanager`.